dss/util/security.py

```diff
@@ -105,27 +106,15 @@ def assert_authorized_group(group: typing.List[str], token: dict) -> None:
     raise DSSForbiddenException()
 
 
-def authorized_group_required(groups: typing.List[str]):
+def assert_security(groups: typing.List[str]):
     def real_decorator(func):
         @functools.wraps(func)
         def wrapper(*args, **kwargs):
             assert_authorized_group(groups, request.token_info)
+            authz_handler = AuthHandler()
+            authz_handler.security_flow(authz_methods=['groups'], groups=groups, token=request.token_info)
```

Comment:
It looks like the `authz_methods` parameter here is supposed to specify in further detail how the Auth provider (e.g., Fusillade) should do  the security flow, but it isn't clear how it's supposed to be used. If this is going to stick around we should add comments/docstrings explaining that parameter.